pkg/3scale/amp/auto-generated-templates/amp/amp-eval-s3.yml

```diff
@@ -208,6 +208,62 @@ objects:
         type: ""
   status:
     dockerImageRepository: ""
+- apiVersion: v1
```

Comment:
Template changes that you'll observe should only be a change of order of the 'redis-config' CR object if I've implemented it correctly (due to the splitting of redis component in several components)